packages/vbdec.vm/vbdec.vm.nuspec

```diff
@@ -2,9 +2,9 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>vbdec.vm</id>
-    <version>12.7.22</version>
+    <version>2.22.23</version>
     <authors>vbGamer45, David Zimmer</authors>
-    <description>VBDec is a VB6 disassembler and PCode debugger</description>
+    <description>VBDec works as a VB6 disassembler and PCode debugger.</description>
```

Comment:
also a structure viewer for all vb6 executables (not just pcode) and can generate IDA scripts to integrate structures and named function offsets (native and pcode)

Reply:
Thanks for the input @dzzie, but as mentioned in [our documentation](https://github.com/mandiant/VM-Packages/wiki/Package-Structure#description), we want to keep the description as short as possible and not to modify it with version updates:

> The description should be short and not include version specific details or other information that is likely to change in a future version.

The description is a nuspec required field, that's why we include, but we are not really using it (as these packages are mostly installed by a installer and are not in the chocolatey community feed). So we would like to save maintaince work.